SnapWar/ViewControllers/SWWarViewController.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+class SWWarViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view?.backgroundColor = UIColor.white
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        navigationController?.setNavigationBarHidden(false, animated: false)
+        
+        navigationController?.navigationBar.topItem?.rightBarButtonItem =
+        UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(onAdd))
+        
+        navigationController?.navigationBar.topItem?.title = "War"
```

Comment:
Thinking we should store VC metadata in a single file that would contain
```
title
activeIcon
inactiveIcon
```

Reply:
we could, but since there's only 3 view controllers that have these special properties, probably not worth the effort tbh